core/cmd/client.go

```diff
@@ -467,7 +467,7 @@ func (s *server) runTLS(port uint16, certFile, keyFile string, writeTimeout time
 	s.lggr.Infof("Listening and serving HTTPS on port %d", port)
 	s.tlsServer = createServer(s.handler, port, writeTimeout)
 	err := s.tlsServer.ListenAndServeTLS(certFile, keyFile)
-	return errors.Wrap(err, "failed to run TLS server")
+	return errors.Wrap(err, "failed to run TLS server (NOTE: you can disable TLS server completely and silence these errors by setting TLSPort=0 in your config)")
```

Comment:
Oh but it is `CHAINLINK_TLS_PORT` in legacy and `WebServer.TLS.HTTSPort` in TOML:
https://github.com/smartcontractkit/chainlink/blob/095a8b30c54a361a5c26b7921a1badc8e7071773/core/config/v2/docs/core.toml#L205-L206